### PR TITLE
Typo and URL fix

### PR DESCRIPTION
**Pull request by gforcada**
Fixes: #

Improves:

Changes proposed in this pull request:
## 
## 
##

### CODE REVIEW

File: develop/plone/serving/traversing.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 .. admonition:: Description
 
-    Plone content is organized to a tree. Traversing means looking up
+    Plone content is organized on a tree. Traversing means looking up
     content from this tree by path. When HTTP request hits a Plone
```
Comment (svx) [inline develop/plone/serving/traversing.rst:8]: I am not a native English speaker, so I am not sure, for me both meaning 'to' and 'on' are sounding weird, at least in the context here, should it be 'as' or 'in' like 'Plone content is organized as/in a tree' ?

Like I said I'm not sure, too :)

**Comment by svx** [comment]
And of course this also should go into the 5.1 branch too !

And btw, thank you at lot, but would you mind using the 'pull request template' next time ? :)

Here is how these could look like:

Fixes: Typo and wrong/old link

Improves: -

Changes proposed in this pull request: - 

Thank you !

File: develop/plone/serving/traversing.rst (hunk 1 above)
Comment (polyester) [inline develop/plone/serving/traversing.rst:8]: my vote for "as"

**Comment by gforcada** [comment]
I changed to `as` then.

**Comment by svx** [comment]
Thanks ! Would you mind to rebase or squash that into one commit? I guess than it would be easier to merge that into 5.0 and 5.1 right ?

**Comment by gforcada** [comment]
@svx rebased, should I merge it right away to both branches?

**Comment by svx** [comment]
If you would like to do that that would be nice ! Yeah it should go into 5.0 and 5.1 

Thanks a lot !!!

**Comment by gforcada** [comment]
Done

**Pull request merged**